snapshot/git/gitdb/db.go

```diff
@@ -0,0 +1,155 @@
+package gitdb
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/scootdev/scoot/os/temp"
+	"github.com/scootdev/scoot/snapshot"
+	"github.com/scootdev/scoot/snapshot/git/repo"
+)
+
+// valueKind describes the kind of a Value: is it a Snapshot or a SnapshotWithHistory?
+// kind instead of type because type is a keyword
+type valueKind int
```

Comment:
We could, but then everywhere we have to do string comparisons, which are harder. So I'd rather do the translation once (on the way in) then having to check each time.